pwa/src/templates/templateParts/layerAccordion/LayerAccordionTemplate.tsx

```diff
@@ -31,7 +32,7 @@ const LayerAccordionTemplate: React.FC<LayerAccordionTemplateProps> = ({ open, s
         <div className={styles.items}>
           {categories.map((category) => (
             <Button
-              className={styles[_.camelCase(`${title} category`)]}
+              className={clsx(styles[_.camelCase(`${title} category`)], styles.categoryButton)}
```

Comment:
This would render `titleToCamelCase category`, right?

What does `category` do here?